crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -199,9 +200,14 @@ pub struct PreviousViewData {
     pub view_proj: Mat4,
 }
 
+#[cfg(not(feature = "meshlet"))]
+type PreviousViewFilter = (With<Camera3d>, With<MotionVectorPrepass>);
+#[cfg(feature = "meshlet")]
+type PreviousViewFilter = Or<(With<Camera3d>, With<ShadowView>)>;
```

Comment:
How come this one has `ShadowView`?

Reply:
ShadowViews do not have Camera3d, but specifically for meshlets I need the previous view for them.